packages/fast-usdc/src/types.ts

```diff
@@ -98,12 +99,22 @@ export interface PendingTx extends CctpTxEvidence {
 }
 
 export type FeeConfig = {
-  /** flat fee charged for every advance */
+  /** flat fee charged for every advance, eligible for LP disbursement */
   flat: Amount<'nat'>;
-  /** proportion of advance kept as a fee */
+  /** proportion of advance kept as a fee, eligible for LP disbursement */
   variableRate: Ratio;
-  /** proportion of fees that goes to the contract (remaining goes to LPs) */
+  /** proportion of `flat` and `variableRate` fees that goes to the contract (remaining goes to LPs) */
   contractRate: Ratio;
+  /**
+   * if present, a fee required to relay (e.g. CCTP to EVM/Solana). Not
+   * considered for LP disbursement - goes to `contractSeat`.
+   */
+  relay?: Amount<'nat'>;
+  /** Optional destination-specific overrides */
+  destinationOverrides?: Record<
+    CaipChainId,
```

Comment:
I take it we'll never want to override based on a particular destination account. Or to override based on a destination namespace.

Reply:
> on a particular destination account

Probably not.

> destination namespace

Maybe - when that requirement comes, I think the current approach is amendable to it.